src/Workspaces/Remote/ServiceHub/Shared/ServiceHubServiceBase.cs

```diff
@@ -46,6 +47,26 @@ protected ServiceHubServiceBase(Stream stream, IServiceProvider serviceProvider)
             Rpc.Disconnected += OnRpcDisconnected;
         }
 
+        protected ServiceHubServiceBase(IServiceProvider serviceProvider, Stream stream)
```

Comment:
see https://github.com/dotnet/roslyn/issues/16900#issuecomment-277378950 if want to know why. and you can ask @AArnott and their team why. I am following their suggestion.